app/gui/view/component-browser/searcher-list-panel/src/lib.rs

```diff
@@ -675,10 +626,19 @@ impl component::Frp<Model> for Frp {
                 &frp_api.input.set_local_scope_section,
                 &frp_api.input.set_sub_modules_section,
             );
-            recompute_layout <- all(&content_update,&layout_update);
+            recompute_layout <- all(&content_update,&layout_frp.update);
             eval recompute_layout(((_,layout)) model.recompute_layout(layout) );
+
+            eval_ model.scroll_area.viewport( model.update_scroll_viewport() );
+
+            is_hovered <- app.cursor.frp.scene_position.map(f!((pos) model.is_hovered(pos.xy())) ).on_change();
```

Comment:
The scene position not always reflects the position in Component Browser space, because the scene may be zoomed out while CB is not. We should take screen position and translate it with `screen_to_object_space` method.